src/frontend/js/app/App.js

```diff
@@ -94,16 +60,12 @@ App.propTypes = {
     user: PropTypes.object.isRequired,
 };
 
-function mapStateToProps(state) {
-    return {
-        user: state.user,
-    };
-}
+const mapStateToProps = state => ({
+    user: state.user,
+});
 
-function mapDispatchToProps(dispatch) {
-    return bindActionCreators({
-        signOut: signOutActions.request,
-    }, dispatch);
-}
+const mapDispatchToProps = dispatch => bindActionCreators({
```

Comment:
I remember something about this yes. However, I had to add it on another component (can't remember which one) otherwise, the dispatch was not injected.